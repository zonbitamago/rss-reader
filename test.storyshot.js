import initStoryshots from "@storybook/addon-storyshots";
import mockdate from "mockdate";
// import { mount } from "enzyme";
import { createMount } from "@material-ui/core/test-utils";

mockdate.set(new Date(2017, 5, 15, 7, 12, 5));
initStoryshots({
  // renderer: mount
  // Portalをサポートできていないため、material-uiのmountを利用する。
  // https://github.com/mui-org/material-ui/issues/9243
  renderer: createMount
});
