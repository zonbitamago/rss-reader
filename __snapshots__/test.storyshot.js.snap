// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Atoms/Button No 1`] = `[Function]`;

exports[`Storyshots Atoms/Button Yes 1`] = `[Function]`;

exports[`Storyshots Atoms/DomainIcon DomainIcon 1`] = `[Function]`;

exports[`Storyshots Atoms/Icon/Header Home 1`] = `[Function]`;

exports[`Storyshots Atoms/Icon/Header RssFeed 1`] = `[Function]`;

exports[`Storyshots Atoms/Icon/Header Twitter 1`] = `[Function]`;

exports[`Storyshots Atoms/Icon/Side Help 1`] = `[Function]`;

exports[`Storyshots Atoms/Icon/Side IndeterminateCheckBox 1`] = `[Function]`;

exports[`Storyshots Atoms/Icon/Side Power 1`] = `[Function]`;

exports[`Storyshots Atoms/Icon/Side Refresh 1`] = `[Function]`;

exports[`Storyshots Atoms/Icon/Side RssFeed 1`] = `[Function]`;

exports[`Storyshots Atoms/Icon/Side Settings 1`] = `[Function]`;

exports[`Storyshots Atoms/Input Name 1`] = `[Function]`;

exports[`Storyshots Atoms/Input URL 1`] = `[Function]`;

exports[`Storyshots Atoms/Input UpdateDuration 1`] = `[Function]`;

exports[`Storyshots Atoms/RegistedListItem RegistedListItem 1`] = `[Function]`;

exports[`Storyshots Atoms/Time 現在日時 1`] = `[Function]`;

exports[`Storyshots Molecules/Item Item 1`] = `[Function]`;

exports[`Storyshots Molecules/Menus Side Menu 1`] = `[Function]`;

exports[`Storyshots Organisms/Header Header 1`] = `[Function]`;

exports[`Storyshots Organisms/RegistedListModal open 1`] = `[Function]`;

exports[`Storyshots Organisms/SettingModal open 1`] = `[Function]`;

exports[`Storyshots Organisms/Side Side 1`] = `[Function]`;

exports[`Storyshots Pages/Page Page 1`] = `[Function]`;
